### PR TITLE
docs: add "Cloud Platform Connection" section

### PR DESCRIPTION
**Pull request by mattdibi**
Add "Cloud Platform Connection" section. This is a simple port of the old documentation with some style fixes and updated images.

### CODE REVIEW

**Comment by mattdibi** [comment]
@marcellorinaldo Can you take a look at this?

File: mkdocs.yml [hunk 1]
```diff
@@ -27,6 +27,12 @@ nav:
       - SSL Configuration: gateway-configuration/ssl-configuration.md
       - Web Console Configuration: gateway-configuration/web-console-configuration.md
       - Gateway Administration Console Authentication: gateway-configuration/gateway-administration-console-authentication.md
+    - Cloud Platforms Connection:
```
Comment (marcellorinaldo) [inline mkdocs.yml]: I would put this section after "Cloud Connection API and Usage:"

File: docs/cloud-platform/kura-ec-cloud.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+The Kura Gateway Administrative Console exposes all services necessary for connecting to Everyware Cloud. The reference links listed below outline each service involved in the Everyware Cloud connection. It is recommended that each section be reviewed.
+
+- [CloudService](#cloudservice)
+- [DataService](}#dataservice)
+- [MqttDataTransport](#mqttdatatransport)
+
+### CloudService
+
+The default settings for the CloudService are typically adequate for connecting to Everyware Cloud. The screen capture shown below displays the default settings for the CloudService. For details about each setting, please refer to [CloudService](#cloudservice).
```
Comment (marcellorinaldo) [inline docs/cloud-platform/kura-ec-cloud.md]: This CloudService link maybe refers to `../cloud-api/built-in-cloud.md`

File: docs/cloud-platform/kura-ec-cloud.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+![](images/cloudService.png)
+
+!!! warning
+    The "*Simple JSON*" payload encoding is not supported by Everyware Cloud. Use the default "*Kura Protobuf*" encoding instead.
+
+### DataService
+
+The majority of default settings in the DataService can be left unchanged. A screen capture of the DataService configuration is shown below. For complete details about the DataService configuration parameters, please refer to [DataService](#dataservice).
```
Comment (marcellorinaldo) [inline docs/cloud-platform/kura-ec-cloud.md]: Same as above, this DataService link maybe refers to ../cloud-api/built-in-cloud.md

File: docs/cloud-platform/kura-ec-cloud.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+- **broker-url** - defines the MQTT broker URL that was provided when the Eurotech Everyware Cloud account was established. Information on how to obtain the broker URL can be found [here](http://everywarecloud.eurotech.com/doc/ECDevGuide/latest/2.02-Managing-Cloud-Users.asp). In the MqttDataTransport configuration screen capture shown below, the *broker-url* is `mqtt://broker-sbx.everyware.io:1883`
+- **topic.context.account-name** - defines the account name of the account to which the device is attempting to connect. In the MqttDataTransport configuration screen capture shown below, the *account name* is `account-name`
+- **username** - identifies the user to be used when creating the connection. In the MqttDataTransport configuration screen capture shown below, the *username* is `username`.
+
+!!! note
+    When connecting to Everyware Cloud, the *username* must have proper permissions. Information on users and permissions can be found [here](http://everywarecloud.eurotech.com/doc/ECDevGuide/latest/2.02-Managing-Cloud-Users.asp).
+
+For complete details about the MqttDataTransport configuration parameters, please refer to [MqttDataTransport](#mqttdatatransport).
```
Comment (marcellorinaldo) [inline docs/cloud-platform/kura-ec-cloud.md]: Same as above, this Data Transport link maybe refers to ../cloud-api/built-in-cloud.md

File: docs/cloud-platform/kura-kapua.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+- [CloudService](#cloudservice)
+
+- [DataService](#dataservice)
+
+- [MqttDataTransport](#mqttdatatransport)
+
+### CloudService
+
+The default settings for the CloudService are typically adequate for connecting to a Kapua instance. The screen capture shown below displays the default settings for the CloudService. For details about each setting, please refer to [CloudService](#cloudservice).
```
Comment (marcellorinaldo) [inline docs/cloud-platform/kura-kapua.md]: This CloudService link maybe refers to `../cloud-api/built-in-cloud.md`

File: docs/cloud-platform/kura-kapua.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+![](images/cloudService.png)
+
+!!! warning
+    The "*Simple JSON*" payload encoding is not supported by Kapua. Use the default "*Kura Protobuf*" encoding instead.
+
+### DataService
+
+The majority of default settings in the DataService can be left unchanged. A screen capture of the DataService configuration is shown below. For complete details about the DataService configuration parameters, please refer to [DataService](#dataservice).
```
Comment (marcellorinaldo) [inline docs/cloud-platform/kura-kapua.md]: This DataService link maybe refers to `../cloud-api/built-in-cloud.md`

File: docs/cloud-platform/kura-kapua.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+### MqttDataTransport
+
+While the majority of default settings in the MqttDataTransport can be left unchanged, the following parameters must be modified:
+
+- **broker-url** - defines the MQTT broker URL that was provided when the Kapua account was established.
+- **topic.context.account-name** - defines the account name of the account to which the device is attempting to connect. In the MqttDataTransport configuration screen capture shown below, the *account name* is `account-name`
+- **username** - identifies the user to be used when creating the connection. In the MqttDataTransport configuration screen capture shown below, the *username* is `username`.
+
+For complete details about the MqttDataTransport configuration parameters, please refer to [MqttDataTransport](#mqttdatatransport).
```
Comment (marcellorinaldo) [inline docs/cloud-platform/kura-kapua.md]: This DataTransport link maybe refers to `../cloud-api/built-in-cloud.md`

File: docs/cloud-platform/kura-hono.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+The reference links listed below outline each service involved in the cloud connection. It is recommended that each section be reviewed.
+
+- [CloudService](#cloudservice)
+- [DataService](#dataservice)
+- [MqttDataTransport](#mqttdatatransport)
+
+### CloudService
+
+The default settings for the CloudService are typically adequate for connecting to a Hono instance. The screen capture shown below displays the default settings for the CloudService. For details about each setting, please refer to [CloudService](#cloudservice).
```
Comment (marcellorinaldo) [inline docs/cloud-platform/kura-hono.md]: This CloudService link maybe refers to `../cloud-api/built-in-cloud.md`

File: docs/cloud-platform/kura-hono.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+![](images/honoCloudService.png)
+
+### DataService
+
+The majority of default settings in the DataService can be left unchanged. A screen capture of the DataService configuration is shown below. For complete details about the DataService configuration parameters, please refer to [DataService](#dataservice).
```
Comment (marcellorinaldo) [inline docs/cloud-platform/kura-hono.md]: This DataService link maybe refers to `../cloud-api/built-in-cloud.md`

File: docs/cloud-platform/kura-hono.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+### MqttDataTransport
+
+While the majority of default settings in the MqttDataTransport can be left unchanged, the following parameters must be modified:
+
+- **broker-url** - defines the MQTT broker URL that was provided when the Eurotech Everyware Cloud account was established. In the MqttDataTransport configuration screen capture shown below, the *broker-url* is `mqtt://broker-url:1883`
+- **topic.context.account-name** - defines the account name of the account to which the device is attempting to connect. In the MqttDataTransport configuration screen capture shown below, the *account name* is `account-name`
+- **username** - identifies the user to be used when creating the connection. In the MqttDataTransport configuration screen capture shown below, the *username* is `username`.
+
+
+For complete details about the MqttDataTransport configuration parameters, please refer to [MqttDataTransport](#mqttdatatransport).
```
Comment (marcellorinaldo) [inline docs/cloud-platform/kura-hono.md]: This TransportService link maybe refers to `../cloud-api/built-in-cloud.md`

**Review by marcellorinaldo — CHANGES_REQUESTED** [review]

**Comment by mattdibi** [comment]
@marcellorinaldo Broken links should be fixed now

**Review by marcellorinaldo — APPROVED** [review]

**Pull request merged**